fixtures/test-server-auth-multiple-tokens/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+auth_tokens="rustypasteisawesome token1 token2 token4"
```

Comment:
I think we should use a bash array here:

```bash
auth_tokens=("rustypasteisawesome" "token1" "token2" "token4")
```

Reply:
Sure, but why? It doesn't give me any advantage. I am just curious.

I love arrays. The reason why I am careful with them is that macOS comes with an ancient version of bash (3.something). I am using a MacPorts bash, but with newer macOS it is even harder to replace `/bin/bash` even with SIP off.
Thus I usually use `#!/usr/bin/env bash`, but sometimes I forget and use `#!/bin/bash` - and then all hell breaks lose. ;-)
Unless I need associative arrays or want to do fancy array stuff, I just use the above used construct.

But I am more than happy to change it. One array coming up.

Reply:
done